packages/frontend-2/components/pricingTable/Plan.vue

```diff
@@ -223,8 +223,12 @@ const isCurrentPlan = computed(() => {
 const isAnnualToMonthly = computed(() => {
   return (
     !isMatchingInterval.value &&
-    props.currentPlan?.name === props.plan &&
-    !isYearlyIntervalSelected.value
+    !isYearlyIntervalSelected.value &&
```

Comment:
I need to revisit some of this logic post release, but dont want make too many changes here last moment.